extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -382,6 +384,34 @@ export class ColumnsTable extends ModelViewBase implements IDataComponent<Predic
 		return warningButton;
 	}
 
+	public async openDialog(title: string, dialogName?: string, width?: azdata.window.DialogWidth, dialogStyle?: azdata.window.DialogStyle, dialogPosition?: azdata.window.DialogPosition) {
```

Comment:
based on current usage, I think you don't have to expose the width, style... parameters.

Reply:
Good call. I will remove all but the properties needed for the callout content.